frontend/src/app/supportAdmin/UnarchivePatients/UnarchivePatient.test.tsx

```diff
@@ -233,44 +232,20 @@ describe("Unarchive patient", () => {
 });
 
 const searchByOrgAndFacility = async () => {
-  await selectDropdown("Organization *", mockOrg1.name);
-  await waitFor(async () =>
-    expect(screen.getByLabelText("Testing facility *")).toHaveTextContent(
-      "Mars Facility"
-    )
+  await selectComboBoxOption("org-selection-container", mockOrg1.name);
+  const [facilityInput] = getFacilityComboBoxElements();
+  await waitFor(async () => expect(facilityInput).toBeEnabled());
+  await selectComboBoxOption(
+    "facility-selection-container",
+    mockFacility1.name
   );
-  await selectDropdown("Testing facility *", mockFacility1.name);
   await clickSearch();
-  await waitFor(() =>
-    expect(screen.queryByText("Loading...")).not.toBeInTheDocument()
-  );
-};
-
-export const clickSearch = async () => {
-  await act(async () => await userEvent.click(screen.getByText("Search")));
-};
-
-export const selectDropdown = async (label: string, value: string) => {
-  await act(
-    async () =>
-      await userEvent.selectOptions(screen.getByLabelText(label), value)
-  );
-};
-
-export const checkPatientResultRows = () => {
-  let resultsRow = screen.getAllByTestId("sr-archived-patient-row");
-  resultsRow.forEach((row, index) => {
-    if (index === 0) {
-      expect(within(row).getByText("Gutmann, Rod")).toBeInTheDocument();
-      expect(within(row).getByText("05/19/1927")).toBeInTheDocument();
-      expect(within(row).getByText("All facilities")).toBeInTheDocument();
-    } else if (index === 1) {
-      expect(within(row).getByText("Mode, Mia")).toBeInTheDocument();
-      expect(within(row).getByText("11/20/1973")).toBeInTheDocument();
-      expect(within(row).getByText("Mars Facility")).toBeInTheDocument();
-    }
+  expect(mockNavigate).not.toHaveBeenCalled();
```

Comment:
we actually removed this check here 😅 https://github.com/CDCgov/prime-simplereport/pull/6479/files#r1319802312 my bad!